pallets/runtime/develop/src/fee_details.rs

```diff
@@ -51,22 +51,32 @@ impl CddAndFeeDetails<AccountId, Call> for CddHandler {
     /// throughout the transaction. This function can also be used to simply check CDD and update identity context.
     fn get_valid_payer(
         call: &Call,
-        caller: &Signatory<AccountId>,
+        caller: &AccountId,
     ) -> Result<Option<AccountId>, InvalidTransaction> {
+        let missing_id = || {
+            Err(InvalidTransaction::Custom(
+                TransactionError::MissingIdentity as u8,
+            ))
+        };
+        let handle_multisig = |multisig, caller: &AccountId, msg| {
+            sp_runtime::print(msg);
```

Comment:
I think we can get rid of this. It's leftover from an ancient debugging attempt :). We have better ways to log stuff now.

Reply:
Done.